jnosql-dynamodb/pom.xml

```diff
@@ -34,6 +34,14 @@
             <groupId>org.eclipse.jnosql.mapping</groupId>
             <artifactId>jnosql-mapping-key-value</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jnosql.communication</groupId>
+            <artifactId>jnosql-communication-document</artifactId>
```

Comment:
Please, remove this dependency. The document already has it.

Reply:
Fixed!